src/platform_impl/macos/app_delegate.rs

```diff
@@ -1,9 +1,23 @@
-use super::app_state::AppState;
+use crate::{platform::macos::ActivationPolicy, platform_impl::platform::app_state::AppState};
+
 use cocoa::base::id;
 use objc::{
     declare::ClassDecl,
     runtime::{Class, Object, Sel},
 };
+use std::{
+    cell::{RefCell, RefMut},
+    os::raw::c_void,
+};
+
+static AUX_DELEGATE_STATE_NAME: &str = "auxState";
+
+pub struct AuxDelegateState {
```

Comment:
I think I would have just put this into the existing `Handler`/`HANDLER` state at some point, for example during `EventLoop::run_return` - but this works as well, so I don't have a strong preference here

Reply:
I'm not a big fan of the`HANDLER` global; everything has to be Atomic or Mutex there.